From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Thu, 7 Jul 2022 17:46:05 +0300
Subject: [PATCH] backport: Avoid non-existing nfo:language on libav extractor

Suppose this should have been dc:language but since to my knowledge
this has never worked, and the other audio/video extractor don't
use such a property, let's just remove this here.
---
 src/tracker-extract/tracker-extract-libav.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/tracker-extract/tracker-extract-libav.c b/src/tracker-extract/tracker-extract-libav.c
index d2d4baf6d68a9dd4ce475dabeea2a2d0e70322d2..87fe6e026a067de4ad8b01fad93a6d7f2e26ab1a 100644
--- a/src/tracker-extract/tracker-extract-libav.c
+++ b/src/tracker-extract/tracker-extract-libav.c
@@ -251,10 +251,6 @@ tracker_extract_get_metadata (TrackerExtractInfo  *info,
 		tracker_resource_set_string (metadata, "nfo:genre", tag->value);
 	}
 
-	if ((tag = av_dict_get (format->metadata, "language", NULL, 0))) {
-		tracker_resource_set_string (metadata, "nfo:language", tag->value);
-	}
-
 	if ((tag = av_dict_get (format->metadata, "title", NULL, 0))) {
 		title = tag->value;
 	}
